lib/cisco_node_utils/cmd_ref/overlay_global.yaml

```diff
@@ -3,7 +3,7 @@
 _exclude: [ios_xr]
 
 anycast_gateway_mac:
-  _exclude: [N3k]
+  _exclude: [N3k, N7k]
```

Comment:
Seems to be an issue with `n7k-j`.  `feature-set fabric` is apparently installed but isn't allowed to be enabled

```bash
n7k-j(config)# show feature-set
Feature Set Name      ID        State
--------------------  --------  --------
fcoe                   1          uninstalled
fabricpath             2          disabled
fex                    3          enabled
mpls                   4          uninstalled
fabric                 7          installed

n7k-j(config)# feature-set ?
  fabricpath  Installed feature-set
  fex         Installed feature-set

n7k-j(config)# feature-set fabric

n7k-j(config)# show feature-set
Feature Set Name      ID        State
--------------------  --------  --------
fcoe                   1          uninstalled
fabricpath             2          enabled
fex                    3          enabled
mpls                   4          uninstalled
fabric                 7          installed
```